acc/components/samples/isotropic_box.py

```diff
@@ -0,0 +1,100 @@
+# -*- python -*-
+#
+# Jiao Lin <jiao.lin@gmail.com>
+#
+
+import numpy as np
+from numba import cuda, void, int64
+from numba.cuda.random import xoroshiro128p_uniform_float32, xoroshiro128p_type
+from math import sqrt, exp
+
+from .SampleBase import SampleBase
+from ...neutron import absorb, prop_dt_inplace
+from ...geometry.onbox import cu_device_intersect_box
+from ...kernels.isotropic import scatter
+
+from ...config import get_numba_floattype
+NB_FLOAT = get_numba_floattype()
+
+
+category = 'samples'
+class IsotropicBox(SampleBase):
+
+    def __init__(
+            self, name,
+            xwidth, yheight, zthickness,
+            mu, sigma,
+    ):
+        """
+        Initialize the isotropicbox component.
+
+        Parameters:
+        name (str): the name of this component
+        xwidth (m): width
+        yheight (m): height
+        zthickness (m): thickness
+        mu (1/m): inverse absorption length
+        sigma (1/m): inverse scattering length
+        """
+        self.name = name
+
+        # Note the configuration of the slit.
```

Comment:
Does it have a slit?

Reply:
A sample does not need a slit in it.

Reply:
just saw it mentioned in the comment. will remove it